### ISSUE TITLE
Infinite loop using e(); in the view

### ISSUE THREAD

**Issue by huglester** [fuel/core#266]
Sorry for hijacking earlier thread ( #263 ),  I was starting the new issue, then thought it's the same, and just copier everything over there :) 

The problem somehow happens, if I try to use Controller_Template, and include e() directly in the view:

``` php
        $var = "<p>test</p>";
        \Debug::dump($var);
        $var = e($var, ENT_COMPAT, \Fuel::$encoding, false);
        \Debug::dump($var);
```

I happen to get errors like this: http://d.pr/soxt now. ( I see around 15 times message about: 
APPPATH/modules/home/views/index.php @ line: 5 )

With a help of @WanWizard, I got to the: https://github.com/fuel/core/commits/develop/classes/security.php
I downloaded 1 st from the top - error
2nd prom the top - error
and with 3rd from top - it starts to work old way ( https://raw.github.com/fuel/core/a284d68b70a06d4822df49ba1bae383528a8f2a3/classes/security.php )

What is bad, that I can't reproduce it under the fresh Fuel install :(

Basically I can remove this e() thing from the view, since I try to make arrays so all data can be cleaned automatically, but others maybe may have similar issuer 

I'll be playing today to see what info I can post more...

Thanks

**Comment by huglester** [comment]
this pull: https://github.com/fuel/core/pull/273 fixes my issue, so I'm closing it

Thanks ! :)
